example/centos/centos-9-x86_64-ami.yaml

```diff
@@ -35,3 +35,6 @@ otk.define.centos-9-x86_64-ami:
       total_size: 10G
 
 otk.include.centos-9-x86_64: "common/centos-9.yaml"
+
+otk.target.osbuild.name:
```

Comment:
Is this a leftover?

Reply:
Yea, dangling leftover. I've removed it.